doc/shared/sunmemory/SUNMemory_Description.rst

```diff
@@ -171,6 +179,33 @@ must define:
    * A new :c:type:`SUNMemory` object.
 
 
+.. c:function:: SUNMemory SUNMemoryHelper_AllocStrided(SUNMemoryHelper helper, \
+                                                       SUNMemory* memptr, \
+                                                       size_t mem_size, size_t stride, \
+                                                       SUNMemoryType mem_type, \
+                                                       void* queue)
+
+   Allocates a ``SUNMemory`` object whose ``ptr`` field is allocated for
+   ``mem_size`` bytes and is of type ``mem_type``. The new object will have
+   ownership of ``ptr`` and will be deallocated when
+   :c:func:`SUNMemoryHelper_Dealloc` is called.
```

Comment:
This appears to be copied from `SUNMemoryHelper_Alloc`, and it's unclear to me if the description needs to be different to account for the `stride` argument.